### PR TITLE
Fixes and basic tooltip usage

### PR DESCRIPTION
**Pull request by ahnlabb**
Still WIP but contains some very important fixes.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -94,7 +94,11 @@ module.exports = {
     filename: '[name]/index.js',
   },
   plugins: [
-    new LodashModuleReplacementPlugin({ shorthands: true }),
+    new LodashModuleReplacementPlugin({
+      shorthands: true,
+      collections: true,
+      paths: true,
+    }),
```
Comment (ErikBjare) [inline webpack.config.js]: What's going on here?

File: src/dashboard/dashboard.js [hunk 2]
```diff
@@ -28,6 +30,14 @@ import 'material-design-icons-iconfont/dist/material-design-icons.css';
   Vue.filter('friendlyShortDuration', formatSecsShort);
   Vue.filter('fixed', (v, precision) => parseFloat(v).toFixed(precision));
 
+  const messages = {
+    tip: {
+      ignore:
+        "This creator won't receive donations and won't show up among your favorite creators",
+    },
+  };
+  Vue.prototype.$t = path => _.get(messages, path);
```
Comment (ErikBjare) [inline src/dashboard/dashboard.js]: What's going on here?

Edit: nvm, noticed `$t` is how things are done by e.g. `vue-i18n`: http://kazupon.github.io/vue-i18n/guide/started.html

File: src/dashboard/components/CreatorCard.vue [hunk 3]
```diff
@@ -29,18 +29,21 @@ v-card(height='116px').mt-1
...
+        v-tooltip(bottom)
+          template(slot='activator')
+            v-list-tile(@click='$emit("ignore")')
+              v-list-tile-action
+                v-icon visibility_off
+              v-list-tile-content
+                v-list-tile-title
+                  | Ignore
+          | {{ $t('tip.ignore') }}
+        v-list-tile(@click='$emit("activity")')
```
Comment (ErikBjare) [inline src/dashboard/components/CreatorCard.vue]: Nice

File: webpack.config.js (hunk 1 above)
Comment (ahnlabb) [reply webpack.config.js]: [lodash-webpack-plugin](https://github.com/lodash/lodash-webpack-plugin) disables certain features to create smaller builds. I needed this particular feature when using `_.get` in `$t`.

**Pull request merged**